<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>s21_interface</class>
 <widget class="QWidget" name="s21_interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLCDNumber" name="lcd_G1">
   <property name="geometry">
    <rect>
     <x>855</x>
     <y>106</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_moveZ_max">
   <property name="geometry">
    <rect>
     <x>1113</x>
     <y>546</y>
     <width>55</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Z++</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_R1">
   <property name="geometry">
    <rect>
     <x>855</x>
     <y>69</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_G3">
   <property name="geometry">
    <rect>
     <x>1182</x>
     <y>106</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>769</x>
     <y>181</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Line Mod</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_Zoom">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>418</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_11">
   <property name="geometry">
    <rect>
     <x>769</x>
     <y>69</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_B3">
   <property name="geometry">
    <rect>
     <x>1182</x>
     <y>143</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_B1">
   <property name="geometry">
    <rect>
     <x>855</x>
     <y>143</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>878</x>
     <y>460</y>
     <width>148</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioButtonCentral">
      <property name="text">
       <string>Central</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonParallel">
      <property name="text">
       <string>Parallel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>933</x>
     <y>183</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Line Size</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>933</x>
     <y>220</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Apex Size</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_moveY">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>504</y>
     <width>64</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>28</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>64</width>
     <height>28</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_Zoom">
   <property name="geometry">
    <rect>
     <x>834</x>
     <y>421</y>
     <width>331</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_R2">
   <property name="geometry">
    <rect>
     <x>1019</x>
     <y>69</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_ApexS">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>220</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_moveX">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>462</y>
     <width>64</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>64</horstretch>
     <verstretch>26</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>28</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>64</width>
     <height>28</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_RotationX">
   <property name="geometry">
    <rect>
     <x>834</x>
     <y>310</y>
     <width>331</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>358</width>
     <height>25</height>
    </size>
   </property>
   <property name="minimum">
    <number>-180</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>765</x>
     <y>13</y>
     <width>102</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Change Color</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>209</y>
     <width>67</width>
     <height>39</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioSolid">
      <property name="text">
       <string>Solid</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioDotted">
      <property name="text">
       <string>Dotted</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>933</x>
     <y>44</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Apex Color</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_32">
   <property name="geometry">
    <rect>
     <x>1096</x>
     <y>106</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_21">
   <property name="geometry">
    <rect>
     <x>933</x>
     <y>69</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_RotationY">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>343</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_22">
   <property name="geometry">
    <rect>
     <x>933</x>
     <y>106</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>769</x>
     <y>44</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Line Color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_moveY_min">
   <property name="geometry">
    <rect>
     <x>1041</x>
     <y>504</y>
     <width>50</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Y--</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>209</y>
     <width>59</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioCube">
      <property name="text">
       <string>Cube</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioLayer">
      <property name="text">
       <string>Layer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioVoid">
      <property name="text">
       <string>Void</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLCDNumber" name="lcd_B2">
   <property name="geometry">
    <rect>
     <x>1019</x>
     <y>143</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>762</x>
     <y>460</y>
     <width>110</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Projection Set:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_moveX_max">
   <property name="geometry">
    <rect>
     <x>1113</x>
     <y>462</y>
     <width>55</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>X++</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_13">
   <property name="geometry">
    <rect>
     <x>769</x>
     <y>143</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_12">
   <property name="geometry">
    <rect>
     <x>769</x>
     <y>106</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_RotationX">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>310</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>378</y>
     <width>58</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Anaxis Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>418</y>
     <width>39</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>348</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Anaxis Y</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_23">
   <property name="geometry">
    <rect>
     <x>933</x>
     <y>143</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_moveZ">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>546</y>
     <width>64</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>28</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>64</width>
     <height>28</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_moveZ_min">
   <property name="geometry">
    <rect>
     <x>1041</x>
     <y>546</y>
     <width>50</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Z--</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_moveX_min">
   <property name="geometry">
    <rect>
     <x>1041</x>
     <y>462</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>X--</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_moveY_max">
   <property name="geometry">
    <rect>
     <x>1113</x>
     <y>504</y>
     <width>54</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Y++</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>762</x>
     <y>281</y>
     <width>259</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Rotation along the coordinate axes</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_RotationY">
   <property name="geometry">
    <rect>
     <x>834</x>
     <y>347</y>
     <width>331</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-180</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_G2">
   <property name="geometry">
    <rect>
     <x>1019</x>
     <y>106</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ScreenShot">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>498</y>
     <width>120</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>ScreeShot</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_LineS">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>188</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="intValue" stdset="0">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_RotationZ">
   <property name="geometry">
    <rect>
     <x>834</x>
     <y>384</y>
     <width>331</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-180</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>835</x>
     <y>181</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Apex Mod</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_LineS">
   <property name="geometry">
    <rect>
     <x>1002</x>
     <y>183</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>310</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Anaxis X</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_R3">
   <property name="geometry">
    <rect>
     <x>1182</x>
     <y>69</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_33">
   <property name="geometry">
    <rect>
     <x>1096</x>
     <y>143</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_31">
   <property name="geometry">
    <rect>
     <x>1096</x>
     <y>69</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_RotationZ">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>378</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1096</x>
     <y>44</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Back Color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Gif">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>498</y>
     <width>120</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>GIF</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_ApexS">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>220</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="intValue" stdset="0">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_openFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
